docs/src/app/(private)/experiments/css-anchor-positioning.module.css

```diff
@@ -0,0 +1,214 @@
+.Anchor {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: black;
+  color: white;
+  width: 8rem;
+  height: 5rem;
+  padding: 0;
+  margin: 0;
+  user-select: none;
+  anchor-name: --trigger;
+}
+
+.Icon {
+  width: 1.25rem;
+  height: 1.25rem;
+}
+
+@position-try --block-end {
+  position-area: block-end;
+}
+@position-try --block-end-start {
+  position-area: block-end span-inline-end;
+}
+@position-try --block-end-end {
+  position-area: block-end span-inline-start;
+}
+@position-try --block-start {
+  position-area: block-start;
+}
+@position-try --block-start-start {
+  position-area: block-start span-inline-end;
+}
+@position-try --block-start-end {
+  position-area: block-start span-inline-start;
+}
+@position-try --inline-start {
+  position-area: inline-start;
+}
+@position-try --inline-start-start {
+  position-area: inline-start span-block-end;
+}
+@position-try --inline-start-end {
+  position-area: inline-start span-block-start;
+}
+@position-try --inline-end {
+  position-area: inline-end;
+}
+@position-try --inline-end-start {
+  position-area: inline-end span-block-end;
+}
+@position-try --inline-end-end {
+  position-area: inline-end span-block-start;
+}
```

Comment:
Don't really know if there's any easier way to ensure center-aligned popups don't overflow their inline axis (aka something close to `shift()` in Floating UI)

```css
position-try-fallbacks:
  --block-end, --block-start-start, --block-start-end, --block-end-start, --block-end-end;
```